src/main/resources/db-scripts/clickhouse/clickhouse.sql

```diff
@@ -7,6 +7,46 @@ DROP TABLE IF EXISTS clinical_event_derived;
 DROP TABLE IF EXISTS genetic_alteration_derived;
 DROP TABLE IF EXISTS generic_assay_data_derived;
 
+-- the following query "fixes" the sample_profile table by adding entrees for "missing" samples -- those which appear in mutated case list but not in the MySQL sample_profile table
+-- this problem was handled in java at run time in legacy codebase
+-- this MUST BE RUN prior to creation of any derived table which relies on sample_profile table
```

Comment:
@sheridancbio note that this MUST be run prior to creation of the derived tables.  Not sure if anything else has sequence dependency in this file